civil-servants.md

```diff
@@ -0,0 +1,50 @@
+# Signalen for civil servants
+
+We're rebuilding Signalen so that municipalities throughout the Netherlands can benefit.
+
+Signalen's advantages include:
+
+* automatic report classification (increasing efficiency because no humans need to be involved)
+* realtime overview of reports, status and history
+* easy integration with other systems through standard APIs (including for external contractors and subcontractors)
+* rich data and analysis opportunities to improve processes
+
+## Operating Signalen
+
+Signalen is unlike any other control center software currently on the market.
+
+### Reliability
+
+Amsterdam's Signalen implementation has a high uptime. We believe an even higher percentage is possible. We're currently rearchitecting Signalen to make it easier to maintain and troubleshoot.
+
+[Read more about Amsterdam's use of Signalen](https://commonground.nl/news/view/54476472/nieuw-meldingensysteem-amsterdam-leert-zelf-van-meldingen) (in Dutch).
+
+### Easy maintenance and support
+
+We're building Signalen to be open source from the start. This allows a flourishing vendor ecology to develop, so that cities have real choice in their procurement and implementation.
+
+### Legal framework
+
+Signalen was built from the start to comply with these laws:
+
+* EU [General Data Protection Regulation](https://eur-lex.europa.eu/eli/reg/2016/679/oj) (GDPR)
+* The Dutch [Archiefwet 1995](https://wetten.overheid.nl/BWBR0007376/2020-01-01)
+
+### Standards compliance and APIs
+
+Signalen is built around common standards and APIs to make it easy to exchange data with other systems.
+
+Signalen already complies with:
+
+* [Common Ground standards](https://commonground.nl/cms/view/54476259/wat-is-common-ground/54476519)
+* CROW (standard main categories and subcategories for machine learning data model classification)
+
+Signalen is easily connected to existing nuisance complaint systems (like [Verbeterdebuurt](https://www.verbeterdebuurt.nl/) and [BuitenBeter](https://www.buitenbeter.nl/).
+
+It already has APIs for connection to common municipal contractors' systems (like Sigmax).
+
+## Learn more about Signalen
+
+This [Common Ground video](https://vimeo.com/389843091/b2d06ebfc0) (9 minutes, in Dutch) provides a full service walkthrough, from citizen-facing frontend to complaint management dashboards, admin consoles and analytics.
+
+Or email <signalen@publiccode.net> to request a demo for your organization.
```

Comment:
For now, let's remove the video until we've gotten permission and the ability to host it elsewhere.

```suggestion
Email <signalen@publiccode.net> to request a demo for your organization.
```